build-conf/CPP.properties

```diff
@@ -0,0 +1,52 @@
+# Releng properties used by language/cc.groovy
+
+#
+# Comma separated list of required build properties for language/cc.groovy
+cpp_requiredBuildProperties=cpp_srcPDS,cpp_objPDS,cpp_loadPDS, \
+cpp_compiler,cpp_linkEditor,cpp_tempOptions,cpp_compileMaxRC
+
+#
+# CC compiler name
+cpp_compiler=CCNDRVR
+
+# CC linker name
+cpp_linkEditor=IEWBLINK
+
+#
+# cc source data sets
+cpp_srcPDS=${hlq}.CPP
+cpp_incPDS=${hlq}.CPP.H
```

Comment:
It's called incPDS, I guess for "include", while all the other references are using "header". Can we have consistency in the way we call "include files" (either includes or headers)?